GoogleDataTransport/GDTCCTLibrary/GDTCCTUploader.m

```diff
@@ -157,104 +158,124 @@ - (nullable NSURL *)serverURLForTarget:(GDTCORTarget)target {
                           p1[27], p2[27], p1[28], p2[28], p1[29], p2[29], p1[30], p2[30], p1[31],
                           p2[31], p1[32], p2[32], p1[33], p2[33], p1[34], p2[34], p1[35], '\0'};
     CSHServerURL = [NSURL URLWithString:[NSString stringWithUTF8String:URL]];
-  });
+
+    static NSURL *INTServerURL;
+    static dispatch_once_t INTOnceToken;
+    dispatch_once(&INTOnceToken, ^{
+      // These strings should be interleaved to construct the real URL. This is just to (hopefully)
+      // fool github URL scanning bots.
+      const char *p1 = "hts/frbslgignenlp.ogepscmv/ieo/eaybtho";
+      const char *p2 = "tp:/ieaeognitra-agolai.o/1frlglgc/aclg";
+      const char URL[77] = {
+          p1[0],  p2[0],  p1[1],  p2[1],  p1[2],  p2[2],  p1[3],  p2[3],  p1[4],  p2[4],  p1[5],
+          p2[5],  p1[6],  p2[6],  p1[7],  p2[7],  p1[8],  p2[8],  p1[9],  p2[9],  p1[10], p2[10],
+          p1[11], p2[11], p1[12], p2[12], p1[13], p2[13], p1[14], p2[14], p1[15], p2[15], p1[16],
+          p2[16], p1[17], p2[17], p1[18], p2[18], p1[19], p2[19], p1[20], p2[20], p1[21], p2[21],
+          p1[22], p2[22], p1[23], p2[23], p1[24], p2[24], p1[25], p2[25], p1[26], p2[26], p1[27],
+          p2[27], p1[28], p2[28], p1[29], p2[29], p1[30], p2[30], p1[31], p2[31], p1[32], p2[32],
+          p1[33], p2[33], p1[34], p2[34], p1[35], p2[35], p1[36], p2[36], p1[37], p2[37], '\0'};
+      INTServerURL = [NSURL URLWithString:[NSString stringWithUTF8String:URL]];
+    });
 
 #if !NDEBUG
-  if (_testServerURL) {
-    return _testServerURL;
-  }
+    if (_testServerURL) {
```

Comment:
Indentation looks wrong, could you please try to fix it with https://github.com/firebase/firebase-ios-sdk#code-formatting

Reply:
This ran through style.sh - but let me check.

Reply:
Thanks, the diff looks so much better now!